.rubocop.yml

```diff
@@ -14,7 +14,6 @@ AllCops:
     - "*/spec/fixtures/**/*"
     - "vendor/**/*"
     - "dry-run/**/*"
-    - "bundler/helpers/v1/patched_bundler"
     - "bundler/helpers/spec_helpers/*"
   NewCops: enable
   TargetRubyVersion: 3.1
```

Comment:
out of scope for this PR, but as an aside, are we really still back on Ruby 3.1 in core? If so, hopefully this removal of bundler v1 will unlock the ability to bump the ruby version as well...

Reply:
I will create a separate issue. After clean-up we can also update the target ruby version.